util/lru_cache.cc

```diff
@@ -256,6 +256,19 @@ Cache::Handle* LRUCacheShard::Lookup(const Slice& key, uint32_t hash) {
   return reinterpret_cast<Cache::Handle*>(e);
 }
 
+bool LRUCacheShard::Ref(Cache::Handle* h) {
+  LRUHandle* handle = reinterpret_cast<LRUHandle*>(h);
+  MutexLock l(&mutex_);
+  if (handle->InCache()) {
+    if (handle->refs == 1) {
```

Comment:
you can turn `handle->refs == 1` into an assert because it should always be true when `handle->InCache() == true`